tests/test_extension.py

```diff
@@ -9,21 +9,6 @@
 from spidermon.contrib.scrapy.extensions import Spidermon
```

Comment:
Let's remove unused imports `Crawler` and `Spider`

Reply:
Fixed!